app/models/user.rb

```diff
@@ -26,7 +26,7 @@ class User < ActiveRecord::Base
   validates_uniqueness_of :name # done by devise
   validates_uniqueness_of :email # done by devise
 
-  validates :joinedwithcode, :presence => true, :inclusion => { :in => $codes, :message => "%{value} is not valid" }, :on => :create
+  validates :joinedwithcode, :presence => true, :inclusion => { :in => User.all.pluck(:code), :message => "%{value} is not valid" }, :on => :create
```

Comment:
aha... no, I don't think we should switch this... there is some kind of situation that I would run into, I forget precisely what it was now, where ActiveRecord hasn't yet been initiated so that this call to the `User` model will work, which is why I had to go directly to a db connection in https://github.com/metamaps/metamaps_gen002/blob/6767771d7d0c49dc6bdacef1a8f126446db7d397/config/initializers/access_codes.rb